client/src/wh/profile/section_company.cljs

```diff
@@ -65,9 +69,14 @@
                                        :text     "View cover letter"
                                        :new-tab? true}])
          (if cv-url
-           [components/underline-link {:href     cv-url
-                                       :text     "View CV"
-                                       :new-tab? true}]
+           [:div (if cv-visible?
+                   [:a {:class styles/underline-link :on-click #(dispatch [::events/close-user-cv])} "Hide cv"]
+                   [:a {:class styles/underline-link :on-click #(dispatch [::events/open-user-cv])} "Show cv"])
```

Comment:
"cv" should be capitalised => "CV"

Reply:
Also, it is possible to condense this code by using a single event - either with "toggle" (e.g. `::events/toggle-show-cv`) semantics or which takes a boolean argument (e.g. `::events/show-user-cv (not cv-visible?)`)